### PR TITLE
fix(Create): floor max tokens to create instead of rounding to arbitrary precision

### PR DESCRIPTION
**Pull request by nicholaspai**
Avoids edge case where `max` sets tokens to a CR === GCR, `Math.floor(x - some_small_amount)` is cleaner.
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

### CODE REVIEW

**Review by chrismaree — COMMENTED** [review]
how does this scale with non-18 decimals? or is this number input here a non-wei delimited so it's 0.0004 synthetic, irrespective of the calling?

**Comment by nicholaspai** [comment]
> how does this scale with non-18 decimals? or is this number input here a non-wei delimited so it's 0.0004 synthetic, irrespective of the calling?

Yeah it's non wei delimited.

File: features/manage-position/Create.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ const Create = () => {
     const maxTokensToCreate = _gcr > 0 ? collateral / _gcr : 0;
     // Unlike the min collateral, we're ok if we round down the tokens slightly as round down
     // can only increase the position's CR and maintain it above the GCR constraint.
-    setTokens(maxTokensToCreate.toFixed(4));
+    setTokens(Math.floor(maxTokensToCreate - 0.00005).toString());
```
Comment (mrice32) [inline features/manage-position/Create.tsx:100]: Why can't we just do something like this instead:
```
setTokens((maxTokensToCreate - 0.0001).toFixed(4));
```

Comment (nicholaspai) [reply features/manage-position/Create.tsx:100]: We can do this

**Review by daywiss — APPROVED** [review]

**Pull request merged**